### PR TITLE
feat: added content to html and css

### PR DESCRIPTION
**Pull request by Itnardoel**
Dejo mi humilde landing page sobre Spiderman.

Top page with a link to watch online
image

About and character section
image

Bottom page with footer and a more section with a buttom to a fandom web
image

### CODE REVIEW

**Review by lyondeveloper — APPROVED** [review]

**Review by lyondeveloper — APPROVED** [review]
me gusto esa implementacion de nuevas cosa de CSS como el media query!

File: Nicolas Pedranti/CSS/styles.css [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+    width: 100vw;
+    height: 50px;
+    margin: -9px 0 0 -9px;
+    position: fixed;
+    z-index: 1;
+}
+
+.nav-list {
+    display: flex;
+    flex-direction: row;
```
Comment (rominasteinberg) [inline Nicolas Pedranti/CSS/styles.css:22]: this line could be ommited (is not incorrect) but row is the default value.

File: Nicolas Pedranti/CSS/styles.css [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+    width: 100vw;
+    margin-left: -9px;
+    margin-bottom: -10px;
+}
+
+footer p {
+    color: white;
+}
+
+@media only screen and (max-width: 1200px) {
```
Comment (rominasteinberg) [inline Nicolas Pedranti/CSS/styles.css:213]: great job using media queries 👍

File: Nicolas Pedranti/CSS/styles.css [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+
+    h1 {
```
Comment (rominasteinberg) [inline Nicolas Pedranti/CSS/styles.css:215]: suggestion: lets group and order by tag, classes and ids. It's not mandatory but helps to read easier the file

**Review by rominasteinberg — COMMENTED** [review]
great job on the name of the commits

**Pull request merged**